<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конвертер валют</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f66867</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gadugi</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff;</string>
     </property>
     <property name="text">
      <string>Конвертер валют</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="input_amount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 30;
color: #fff;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_amount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
border-radius: 30;
color: #fff;
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="output_currency">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Felix Titling</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
color: #fff;</string>
    </property>
    <item>
     <property name="text">
      <string>RUB - Рубль</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USD - Доллар США</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR - Евро</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY - Японская йена</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP - Британский фунт стерлингов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CAD - Канадский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AUD - Австралийский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHF - Швейцарский франк</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SGD - Сингапурский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HKD - Гонконгский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEK - Шведская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOK - Норвежская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DKK - Датская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NZD - Новозеландский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ZAR - Южноафриканский рэнд</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BRL - Бразильский реал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KRW - Южнокорейская вона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CNY - Китайский юань</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INR - Индийская рупия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MYR - Малайзийский ринггит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THB - Тайский бат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MXN - Мексиканское песо</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PHP - Филиппинское песо</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HUF - Венгерский форинт</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IDR - Индонезийская рупия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CZK - Чешская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RON - Румынский лей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRY - Турецкая лира</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISK - Исландская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HRK - Хорватская куна</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BGN - Болгарский лев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NZD - Новозеландский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PLN - Польский злотый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LTL - Литовский лит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SIT - Словенский толар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ROL - Румынский лей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRL - Турецкая лира</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SKK - Словацкая крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LVL - Латвийский лат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MTL - Мальтийская лира</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EEK - Эстонская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CYP - Кипрский фунт</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Felix Titling</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
color: #fff;</string>
    </property>
    <property name="text">
     <string>КОНВЕРТИРОВАТЬ</string>
    </property>
   </widget>
   <widget class="QComboBox" name="input_currency">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>261</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Felix Titling</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e;
border: 2px solid #f66867;
color: #fff;</string>
    </property>
    <item>
     <property name="text">
      <string>RUB - Рубль</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USD - Доллар США</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EUR - Евро</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JPY - Японская йена</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBP - Британский фунт стерлингов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CAD - Канадский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AUD - Австралийский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CHF - Швейцарский франк</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SGD - Сингапурский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HKD - Гонконгский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEK - Шведская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOK - Норвежская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DKK - Датская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NZD - Новозеландский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ZAR - Южноафриканский рэнд</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BRL - Бразильский реал</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KRW - Южнокорейская вона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CNY - Китайский юань</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>INR - Индийская рупия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MYR - Малайзийский ринггит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>THB - Тайский бат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MXN - Мексиканское песо</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PHP - Филиппинское песо</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HUF - Венгерский форинт</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IDR - Индонезийская рупия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CZK - Чешская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RON - Румынский лей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRY - Турецкая лира</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISK - Исландская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HRK - Хорватская куна</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BGN - Болгарский лев</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NZD - Новозеландский доллар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PLN - Польский злотый</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LTL - Литовский лит</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SIT - Словенский толар</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ROL - Румынский лей</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TRL - Турецкая лира</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SKK - Словацкая крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LVL - Латвийский лат</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MTL - Мальтийская лира</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EEK - Эстонская крона</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CYP - Кипрский фунт</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Найдите исходную валюту в списке</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
